zuul.d/jobs.yaml

```diff
@@ -22,6 +22,9 @@
       - LICENSE
       - OWNERS
       - .*/*.md
+    vars:
+      cifmw_set_openstack_containers_overrides:
+        RELATED_IMAGE_OS_CONTAINER_IMAGE_URL_DEFAULT: "quay.io/steveb/edpm-hardened-uefi:18.0.0-5"
```

Comment:
Looks like it did not use the image  ` RELATED_IMAGE_OS_CONTAINER_IMAGE_URL_DEFAULT:  quay.io/podified-antelope-centos9/edpm-hardened-uefi@sha256:12f80f94f00cef35264ec50d113fac9956eea5ae293e4b47f9045d7f2396888e` 

https://logserver.rdoproject.org/47/647/756c34d8b64d6e56d381a139fe355e51122682fb/github-check/install-yamls-crc-podified-edpm-baremetal/7249891/ci-framework-data/logs/quay-io-openstack-k8s-operators-openstack-must-gather-sha256-e213dc67dd35a55602651933d2b9edf330b1da5922998f63fdb145712bbf00fa/namespaces/openstack-operators/pods/openstack-baremetal-operator-controller-manager-7494d9965cfhd9l/openstack-baremetal-operator-controller-manager-7494d9965cfhd9l-describe